test/error_messages.coffee

```diff
@@ -1142,3 +1142,10 @@ test "imported members cannot be reassigned", ->
     export foo = 'bar'
            ^^^
   '''
+
+test "bound functions cannot be generators", ->
+  assertErrorFormat 'f = => yield this', '''
+    [stdin]:1:5: error: bound (fat arrow) functions cannot contain 'yield'
+    f = => yield this
+        ^^^^^^^^^^^^^
```

Comment:
It just underlines whatever the first line of the function is, starting at the `=>`:
```
./bin/coffee -bpe 'f = =>
quote>   1
quote>   2
quote>   yield this'
[stdin]:1:5: error: bound (fat arrow) functions cannot contain 'yield'
f = =>
    ^^
```